app/pages/T7cn0yCTBUmUlBfQq3JfKA.tsx

```diff
@@ -0,0 +1,146 @@
+import { GetServerSideProps } from "next";
+import Head from "next/head";
+
+import { Progress } from "../components/Progress";
+import {
+  DosesDescription,
+  DosesDescriptionList,
+} from "../components/DosesDescriptionList";
+import { DarkModeToggle } from "../components/DarkModeToggle";
+import { DhbPopulationDoseData } from "../types/VaccineDataTypes";
+import fetchHomePageProps from "../services/homePagePropsService";
+import { useHasMounted } from "../hooks/useIsMounted";
+import { RegionDropdown } from "../components/RegionDropdown";
+import { useStickyState } from "../hooks/useStickyState";
+import { DhbRegionId } from "../components/RegionDropdown/RegionDropdown";
+
+export type HomePageProps = {
+  allDhbsPopulationDoseData: DhbPopulationDoseData[];
+  aucklandDhbsPopulationDoseData: DhbPopulationDoseData[];
+  combinedAucklandDhbsPopulationDoseData: DhbPopulationDoseData;
+  dataValidAsAtTimeUtc: string;
+  dataFetchedAtTimeUtc: string;
+};
+
+const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
+  const { aucklandDhbsPopulationDoseData } = props;
+
+  const sortedAucklandDhbsPopulationDoseData =
+    aucklandDhbsPopulationDoseData.sort((a, b) =>
+      a.dhbName.localeCompare(b.dhbName)
+    );
+
+  const hasMounted = useHasMounted();
+
+  return (
+    <div className="h-full min-h-screen">
+      <Head>
+        <meta name="robots" content="noindex" />
```

Comment:
I think we might want?

```suggestion
        <meta name="robots" content="noindex, nofollow" />
        <meta name="googlebot" content="noindex, nofollow" />
```

Reply:
From what I read `googlebot` only blocks google whereas `robots` blocks most/all including Google. `nofollow` is about the crawler not following the links on the page the go to other pages. So may be fine? I can update if we wanna be safe

https://developers.google.com/search/docs/advanced/crawling/block-indexing

Reply:
Made the change any, this is temporary

Reply:
Ah gotcha - I copied and pasted from one I new haha

Feel free to commit or ignore!